includes/Stories_Renderer/Carousel_Renderer.php

```diff
@@ -85,22 +90,42 @@ public function render( array $args = [] ) {
 		ob_start();
 		?>
 		<div class="<?php echo esc_attr( $container_classes ); ?>">
-			<amp-carousel
-				width="1"
-				height="1"
-				layout="intrinsic"
-				type="carousel"
-				role="region"
-				aria-label="<?php esc_attr_e( 'Web Stories', 'web-stories' ); ?>"
-			>
+			<?php
+			if ( ! $this->is_amp_request() ) {
+				?>
+				<div class="web-stories-list__carousel">
+					<?php
+					foreach ( $this->story_posts as $story ) {
+						$this->render_single_story_content();
+						$this->next();
+					}
+					?>
+				</div>
+				<div tabindex="0" aria-label="<?php esc_attr_e( 'Previous', 'web-stories' ); ?>" class="glider-prev">←</div>
+				<div tabindex="0" aria-label="<?php esc_attr_e( 'Next', 'web-stories' ); ?>" class="glider-next">→</div>
```

Comment:
```suggestion
				<div tabindex="0" aria-label="<?php esc_attr_e( 'Next', 'web-stories' ); ?>" class="glider-next">&#8594;</div>
```

Reply:
Curious, how does this work in RTL context? Sounds like the `&#8594;` here should be translatable?

Reply:
Good question. There are places in core that seem to translate this characters  
https://github.com/WordPress/wordpress-develop/blob/master/src/wp-content/themes/twentyeleven/single.php#L20-L24

Reply:
`amp-carousel` doesn't use any text for the carousel arrows, instead it uses background-image which gets swapped for `rtl`, I was thinking to do the same instead.

and also `[glider-js](https://github.com/NickPiscitelli/Glider.js/issues/52)` doesn't support `rtl` just yet. Do we want to reconsider using it?

Reply:
What does the carousel currently look like on RTL vs. LTR?

I assume it would not take much to add RTL support beyond passing the stylesheet through RTLCSS.

Reply:
Slider arrows won't work, but the scroll will work. and styles won't break.

Reply:
Slider arrows won't work at all, or do they just go to the wrong direction?

If they go to the wrong direction, could we do something like this? 

```js
const isRTL = 'rtl' === document.documentElement.getAttribute('dir');

new Glider(carouselWrapper, {
	// ...
    arrows: {
      prev: isRTL ? '.glider-next' : '.glider-prev',
      next: isRTL ? '.glider-prev' : '.glider-next',
    }
	// ...
}
```

Reply:
Arrows won't work at all, I checked if that's possible but the slider doesn't implement the scroll behavior for RTL direction.

Reply:
Dang 😞 

But let's keep glider for now, just so we have a working solution. We can follow-up with RTL support later

Reply:
I'll add a todo for the RTL support for now. this will only be an issue for non-AMP pages.

Reply:
I've extended the glider's `scrollItem` method to add basic support for RTL, it's kind of a trick but will work on RTL, without confusing users.

@swissspidy please take a look at the [carousel/index.js](https://github.com/rtCamp/web-stories-wp/blob/feature/stories-carousel/assets/src/carousel/index.js)

Reply:
@divyarajmasani Oh wow, amazing!